packages/config/config/devices/0x015f/mh-p220.json

```diff
@@ -182,7 +260,29 @@
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Wire with Roller Blinds Switch",
+					"value": 0
+				},
+				{
+					"label": "Enable Scene Activation and send basic set on/off to AG4",
```

Comment:
I think this needs to be even shorter. How about: "Scene activation; Control Association Group 4"